### ISSUE TITLE
Why need define "log_sum_exp" other than use the cross entropy loss in pytorch?

### ISSUE THREAD

**Issue by USTClj** [ShuangXieIrene/ssds.pytorch#32]
I'm a green hand in pytorch. Any help is welcomed. Thank you!

**Comment by Realwhisky** [comment]
The 'log_sum_exp' helps u to find the hardest examples and get a numerical sequence of them.
